src/screens/CreatePost/elements/useHastagMention.js

```diff
@@ -6,14 +6,14 @@ const useHastagMention = (initial = '') => {
   const [formattedText, setFormattedText] = React.useState(initial);
   const [hashtags, setHashtags] = React.useState([]);
 
-  const handleStateHashtag = (text, setHashtagState = null) => {
-    const hashtagMention = handleHastagMention(text, hashtags, setHashtagState);
+  const handleStateHashtag = (text) => {
+    const hashtagMention = handleHastagMention(text, hashtags);
     setFormattedText(hashtagMention);
   };
 
-  const updateHashtag = (text, hashtagsProp, setHashtagState = null) => {
+  const updateHashtag = (text, hashtagsProp) => {
     setHashtags(hashtagsProp);
-    setFormattedText(handleHastagMention(text, hashtagsProp, setHashtagState));
+    setFormattedText(handleHastagMention(text, hashtagsProp));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the above comment, the removal of the `setHashtagState` parameter from the `updateHashtag` function seems to be a good change, as it was not being used in the previous implementation. Again, please ensure that this function is not called elsewhere with this additional argument. 

Overall, these changes seem to improve the readability and maintainability of the code by removing unused parameters.


<!-- This is an auto-generated comment by OSS CodeRabbit -->